content/en/blog/2020/istiocon-2021/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Join us for the first IstioCon in 2021!"
+description: The inaugural conference for Istio will take place at the end of February.
+publishdate: 2020-12-04
```

Comment:
This should be updated to the date when the PR is ready to merge. If it's ready today, then:
```suggestion
publishdate: 2020-12-08
```